### DISCUSSION TITLE
How to handle multi-graph tasks, that is, the training data is (B, N, C)

### DISCUSSION THREAD

**Discussion by hellomuyi** [pyg-team/pytorch_geometric#4697] [Q&A] [answered]
Hello, does PyG support graph-level tasks, such as graph classification, which means that the shape of the input training data should be (batch_size, num_nodes, num_features). As is common practice in DNNs, the batch_size is num_graphs instead of num_nodes. If so, can you give me a link to an example?

**Comment by rusty1s** [comment, marked as answer]
Yes, PyG supports graph-level tasks, but the batching is a little bit different from traditional neural networks, see [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/introduction.html#mini-batches) and [here](https://pytorch-geometric.readthedocs.io/en/latest/notes/batching.html). [Here](https://github.com/pyg-team/pytorch_geometric/blob/2.0.4/examples/mutag_gin.py) is a practical example of a graph-level classification task.
